#goal definition
float64 targetAngle
float64 successThreshold
---
#result definition
bool finished
float64 angleError
---
#feedback
float64 angleError
